### ISSUE TITLE
The number of points  is decreasing when training large-scale scenes.

### ISSUE THREAD

**Issue by xiemeilong** [graphdeco-inria/gaussian-splatting#161]
I am training a large-scale scene, close to eight thousand images. However, the training results are not ideal. I looked at the size of the point_cloud.ply files in the checkpoints, which was over 500 MB after 3000 iterations but only around 90 MB after 9000 iterations. It kept getting smaller after that, ending up at just over 8 MB. 

```
python train.py -s ../data/gs/xingyuanbian2 -i images_4 --data_device cpu \
--position_lr_init 0.0000016 --position_lr_final 0.000000016 --scaling_lr 0.00005 --save_iterations 3000 9000  15000
```

**Comment by ang88myt** [comment]
#Key Points:
Model Instability: The model might be unstable, leading to the point cloud collapsing into a smaller set of points as training progresses.

Overfitting: The model could be overfitting to the training data, causing it to lose its ability to capture the full complexity of the point cloud.

Learning Rate Issues: If the learning rate is too high or too low, the optimization process might not be effective, resulting in a suboptimal point cloud.

#Suggested Actions:
Checkpoint Analysis: Load different checkpoints and analyze their performance on validation data to identify when the model starts to deteriorate.

Hyperparameter Tuning: Adjust the learning rates (--position_lr_init, --position_lr_final, --scaling_lr) and experiment with other hyperparameters.

Loss Monitoring: Closely monitor the loss function and other performance metrics during training. A sudden increase or fluctuation could indicate instability.

Data Sanity Check: Ensure that the input data and initial point cloud are correctly formatted and scaled. Incorrect data could lead to such behavior.

**Comment by xiemeilong** [comment]
@ang88myt Your comment sounds like ChatGPT. Thanks anyway.

**Comment by Snosixtyboo** [comment]
Hi,

this can happen if you get extremely unlucky with your scene files. The algorithm has the potential for a draining effect due to the culling of large gaussians. I will see if i can make a quickfix for it, but in the meantime, you could go and comment this line 
```prune_mask = torch.logical_or(torch.logical_or(prune_mask, big_points_vs), big_points_ws)``` in ```gaussian_model.py``` and that should avoid it

**Comment by xiemeilong** [comment]
The issue  was due to mismatch of images in similar areas. However, large scenes still cannot achieve the same level of quality as small scenes. Taking a small region from a large scene can produce very good results. But points in large scene tend to not split , failing to achieve a detailed effect. Even greatly reducing these two parameters --densify_grad_threshold 0.00008 --percent_dense 0.002 still cannot attain the same quality as small scenes.

**Comment by xiemeilong** [comment]
I will open a new issue.

**Comment by JunhuaLiu0** [comment]
@xiemeilong  Hi, I wonder if the size of point_cloud.ply for large scenes will be larger than for small scenes. I am curious about the file size of the point_cloud.ply by 8000 pictures you mentioned. Could you please provide some data?

Sincerely,
Thank you.
